src/herbivoresAndCarnivores.js

```diff
@@ -6,6 +6,17 @@ class Animal {
     this.health = 100;
     Animal.alive.push(this);
   }
+
+  bite(animal) {
+    if (animal instanceof Herbivore && !animal.hidden) {
+      animal.health -= 50;
+      animal.die();
+    }
```

Comment:
Almost there, just move bite method to Carnivore class. After you do that, you can also move animal.die() outside of if statement.